tests/Unit/Classes/Tax/TaxCalculatorCoreTest.php

```diff
@@ -61,6 +61,6 @@ public function testGetTotalRateBug()
 
         $totalRate = $tax_calculator->getTotalRate();
 
-        $this->assertEquals(27.233, $totalRate);
```

Comment:
Maybe the tests should not be updated, if a lib upgrade breaks tests, we need to update the code, not the tests.

Or perhaps I'm missing something here

Reply:
I agree with @FabienPapet, it would be interesting to dive a little in the code to see if the code or the test should be changed.

Reply:
@matks yesterday we discussed that the code is ok. What's your opinion about this PR please?

Reply:
@nicosomb if you already checked, just give a little bit of detail about why the code is valid in the PR's description so we can better understand :)

Reply:
@nicosomb with such a sensitive topic I would always say "yes, let's double-check" 😄 a little dive in the code is worth doing if we can avoid horrible float computation errors in the future

Reply:
🆙 @nicosomb 😄

Reply:
I could see that if we update only phpunit, tests are failing. So the code looks right here

Reply:
So if code is ok, PR is ok?